packages/lib-user/src/components/MyGroups/components/GroupCard.stories.js

```diff
@@ -0,0 +1,42 @@
+import { Box } from 'grommet'
+
+import GroupCard from './GroupCard.js'
+
+export default {
+  title: 'Components/MyGroups/GroupCard',
+  component: GroupCard,
+  decorators: [ComponentDecorator]
+}
+
+function ComponentDecorator (Story) {
+  return (
+    <Box
+      background={{
+        dark: 'dark-3',
+        light: 'neutral-6'
+      }}
+      pad='30px'
+      width='624px'
+    >
+      <Story />
+    </Box>
+  )
+}
+
+export const Admin = {
+  args: {
+    displayName: 'Group Name',
+    classifications: 1234,
+    contributors: 89,
+    hours: 567,
+    projects: 10,
+    role: 'group_admin'
+  }
+}
```

Comment:
I'd refactor these into a `GroupCard.mock.js` so you can use them in your specs instead of creating the brittle hardcoding between the two files.

Reply:
A good example of this is in the `ConnectWithProject.mock.js` file and how its used in the story and spec files.

Reply:
Great suggestion, updated accordingly 👍